app/src/androidTest/java/org/mozilla/fenix/ui/robots/SyncSignInRobot.kt

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Not sure if i am seeing this correctly, but I think there's an extra space.

Reply:
I can't see one